### PR TITLE
Add many more things

### PR DESCRIPTION
**Pull request by Kaamkiya**
Things I've added:

* AlpineJS
* Deno
* d3js
* NPM million package mark
* NPM initial release

### CODE REVIEW

File: data/2019.ts [hunk 1]
```diff
@@ -13,6 +19,11 @@ export default {
             link: 'https://github.com/vercel/next.js/releases/tag/v9.0.0',
             icon: 'next'
         },
+        {
+            date: '2019-06-04',
+            title: 'NPM passes 1 million packages',
+            link: 'https://snyk.io/blog/npm-passes-the-1-millionth-package-milestone-what-can-we-learn/',
+            icon: 'npm'
```
Comment (michalkuncio) [inline data/2019.ts:26]: Missing '},'

File: data/2020.ts [hunk 2]
```diff
@@ -49,12 +49,23 @@ export default {
...
             date: '2020-04-21',
             title: 'Node.js 14.0 released',
             link: 'https://github.com/nodejs/node/releases/tag/v14.0.0',
             icon: 'node'
         },
+        {
+            date: '2020-02-19',
+            title: 'AlpineJS 2.0 released',
+            link: 'https://github.com/alpinejs/alpine/releases/tag/v2.0.0',
+            icon: 'alpinejs'
```
Comment (michalkuncio) [inline data/2020.ts]: Missing '},'

**Review by michalkuncio — CHANGES_REQUESTED** [review]
Thank you for adding many events! There are some missing curly brackets in the 2019.ts and 2020.ts files. It should be a simple fix.

File: data/2019.ts (hunk 1 above)
Comment (Kaamkiya) [reply data/2019.ts:26]: Oops, sorry! I don't know how I missed that. Fixed now :)

File: data/2020.ts (hunk 2 above)
Comment (Kaamkiya) [reply data/2020.ts]: Again, I don't know how I missed that. Fixed now, though.

**Comment by Kaamkiya** [comment]
I've now fixed those and a couple of broken icons. Should be good to go, but please tell me if there're any other problems with the code I've written.

File: data/2010.ts [hunk 3]
```diff
@@ -12,6 +12,11 @@ export default {
             title: 'Knockout 1.0 released',
             link: 'https://github.com/knockout/knockout/releases/tag/v1.0.0',
             icon: 'knockout'
-        }
+        },
+        {
+            date: '2010-01-12',
+            title: 'NPM is released',
+            link: 'https://www.npmjs.com/',
+            icon: 'npm'
```
Comment (michalkuncio) [inline data/2010.ts:20]: Missing '}'

**Comment by michalkuncio** [comment]
Just one fix and it should be good!

**Comment by Kaamkiya** [comment]
Where's the other fix? Sorry, can you just put the file name and line number so I know what to do? Thanks!

**Review by michalkuncio — CHANGES_REQUESTED** [review]
Missing '}' on line 21 in 2010.ts.

**Comment by Kaamkiya** [comment]
I think it should be good to go now, but if you find anything else, let me know so I can fix it :)

**Review by michalkuncio — APPROVED** [review]
Looks good now! Thank you for contributing!

**Pull request merged**